clean.py

```diff
@@ -0,0 +1,243 @@
+## Import libraries for functions
+import os
+import pathlib
+import earthpy as et
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import rasterstats as rs
+import xarray as xr
```

Comment:
You don't need xr or rxr in this repo

Reply:
whoops, I didn't run flake8 over this notebook - darn that's an easy one to fix!